book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :author, :title, :rentals
+

```

Comment:
- [ ] Kindly create a many-to-many relationship between Person and Book class through rentals. Add a method in the Book class for adding rentals. Also, add a method for adding rentals in the Person class. Check the example [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship).

Reply:
Thank you @Kamba56  for pointing this out. I'll rectify it immediately.

Reply:
Fixed.